### PR TITLE
States/washington + 8 more

### PR DESCRIPTION
**Pull request by delaneypage**
States: 
- Arkansas
- Montana 
- Oregon
- Oklahoma
- Rhode Island 
- South Dakota
- Washington 
- West Virginia
- Wyoming
- Alaska 
- Connecticut 
- Delaware
- Maryland 
- Vermont 
- DC

Comments: 
- No history files for West Virginia, Oregon

Other changes:
- DOWNLOAD.PY line 336: lower() file extension strings in if statement (decompress)
- CONFIGS.PY lines 144-149: fill NaN with pd.NaT to avoid error "float" has no attribute "year" (coerce_dates)

### CODE REVIEW

File: reggie/ingestion/download.py [hunk 1]
```diff
@@ -170,6 +170,7 @@ def __init__(self, config_file=CONFIG_OHIO_FILE, force_date=None,
         self.state = config["state"]
         self.obj_will_download = False
         self.meta = None
+#        self.dataframe = None
```
Comment (cvcotton) [inline reggie/ingestion/download.py:173]: maybe clean up?

File: reggie/ingestion/download.py [hunk 2]
```diff
@@ -399,6 +401,8 @@ def generate_local_key(self, meta=False):
         return name
 
     def output_dataframe(self, file_item):
+        # if self.dataframe is not None:
```
Comment (cvcotton) [inline reggie/ingestion/download.py:404]: clean up

File: reggie/configs/configs.py [hunk 3]
```diff
@@ -21,9 +21,10 @@ def __init__(self, state=None, file_name=None):
         self.data = self.load_data(config_file, self.infer_locale_file(
             config_file))
 
-        self.primary_locale_column = self.data[PRIMARY_LOCALE_ALIAS]
-        self.primary_locale_type = self.data.get(PRIMARY_LOCALE_TYPE, "county")
-        self.primary_locale_names = self.data[PRIMARY_LOCALE_NAMES]
+        if self.data[PRIMARY_LOCALE_NAMES] is not None:
+            self.primary_locale_column = self.data[PRIMARY_LOCALE_ALIAS]
```
Comment (cvcotton) [inline reggie/configs/configs.py:25]: if you switch your county identfiers to primary locale identifers, you can leave these 2 lines out of the conditional then

File: reggie/ingestion/download.py (hunk 1 above)
Comment (delaneypage) [reply reggie/ingestion/download.py:173]: tinkering w/ dataframe output, will clean off my master branch (right now read_csv is called twice, was testing what would happen if we saved the preprocess dataframe as an attribute to return instead, but don't know how that would affect memory management)

**Pull request merged**